app/templates/index.hbs

```diff
@@ -27,31 +27,42 @@
 
   <ul class="ember-version-graphics">
     <li class="item list-unstyled">
-      <EmberVersionGraphic @mascot='tomster-deprecated' @text='1.x' />
+      <EmberVersionGraphic @mascot="ember" />
       <ul class="links">
-        <li class="list-unstyled" data-test-ember-1-link><LinkTo @route='ember' @model='v1.x'>Ember</LinkTo></li>
+        <li class="list-unstyled" data-test-ember-1-link>
+          <LinkTo @route="ember" @model="v1.x">v1.x</LinkTo>
+        </li>
+        <li class="list-unstyled" data-test-ember-1-link>
```

Comment:
```suggestion
        <li class="list-unstyled" data-test-ember-2-link>
```

I'm assuming this number should correspond with the listed version. This is also the case for the other `data-test-ember` attributes.

Reply:
Thanks for reviewing :) Made the changes as suggested